docs/community/tutorials-contest/2022/avalanche-theGraph-index/README.md

```diff
@@ -0,0 +1,381 @@
+
+![graph](./images/21.png "graph")
+
+### Prerequisites
+
+1. NodeJS and Yarn
+
+First, install the LTS (long-term support) version of [nodejs](https://nodejs.org/en). This is `16.2.0` at the time of writing. NodeJS bundles `npm`.
+
+Next, install the [yarn](https://yarnpkg.com) package manager:
+
+```zsh
+npm install -g yarn
+```
+
+2. Git
+
+To check the current Git version use:
+
+```zsh
+git --version
+```
+
+This tutorial is created to serve as a guide to help developers setup an Avalanche Subnet and Index them using graphQl. We are going to learn how to run a local network using the [Avalanche-cli](https://github.com/ava-labs/avalanche-cli) and deploy a basic smart contract using [Remix](https://remix.ethereum.org/). Then Lastly we will be indexing our Subnet using [The Graph](https://thegraph.com/). This guide is an extension of the [Official Avalanche Documentation](https://docs.avax.network/subnets/create-a-local-subnet).
+
+Please note that all command line inputs and sample codes are MacOs and Linux Based. Commands may vary for other operating systems.
+
+In summary, we will be discussing the following:
+1. Running an EVM Subnet on the Local Network using the Avalanche-cli
+2. Deploying smart contracts with Remix
+3. Indexing our Subnet using The Graph
+
+
+### Running an EVM Subnet on the Local Network Using the Avalanche-cli
+
+We will be creating an EVM on our local machine to give us a basic feel on how a Subnet functions. The [Avalanche-CLI](https://github.com/ava-labs/avalanche-cli) is a novel tool that allow us to have a local network up in minutes.
+
+
+1. Installation
+
+Open up you MacOs command line utility and run the following command
+
+On your home directory, create a new directory and `cd <newdir>` into the directory. This is where we will be installing all our project dependencies.
+
+```zsh
+curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-cli/main/scripts/install.sh | sh -s
+```
+
+This command download the latest binary of the [Avalanche-CLI](https://github.com/ava-labs/avalanche-cli) to the current directory where it was executed.
+
+`cd` into the `bin` folder and export the path variable
+
+```zsh
+cd bin
+export PATH=$PWD:PATH
+```
+
+This makes the `avalanche` command available globally. For more information about [environment-variables](https://apple.stackexchange.com/questions/106778/how-do-i-set-environment-variables-on-os-x) and [avalanche-cli-commands](https://docs.avax.network/subnets/create-a-local-subnet#quickstart) visit the respective links.
+
+![variables](./images/cover.jpeg "variables")
+
+
+2. Initialising a Default Subnet
+
+We will be using the `avalanche subnet create` command line wizard to get our network running. ASAP.
+In the same directory where the binary was installed, run the following command
+
+```zsh
+avalanche subnet create <SubnetName>
```

Comment:
[Avalanche CLI v0.2.1](https://github.com/ava-labs/avalanche-cli/releases/tag/v0.2.1) was just released, can you please check that the commands are all still the same (there were some changes)?

Reply:
So far the commands still work